ui/src/holdingpen-new/containers/DetailPageContainer/AuthorDetailPageContainer.tsx

```diff
@@ -25,6 +25,7 @@ import {
   columnsSubject,
   columnsAdvisors,
 } from './columnData';
+import { getConfigFor } from '../../../common/config';
```

Comment:
Please don't forget to add the default value here https://github.com/inspirehep/inspirehep/blob/2e1f29feb410705d7efa8021f5c4cff8a6b4f002/ui/public/config.js otherwise it will fail locally